providers/flagd-in-process/pkg/provider.go

```diff
@@ -59,6 +59,7 @@ type connectionInfo struct {
 
 type connectionState int
 
+// DEPRECATED : Please use flagd with WithInProcessResolver option instead of this dedicated provider
```

Comment:
It should be like the following:
```suggestion
// Deprecated: Please use flagd with WithInProcessResolver option instead of this dedicated provider
```
You can see more info around this here: https://github.com/golang/go/issues/10909